src/types/parse_mode.rs

```diff
@@ -180,6 +185,112 @@ impl FromStr for ParseMode {
     }
 }
 
+impl Markup for ParseMode {
+    fn bold(&self, s: &str) -> String {
+        match self {
+            Self::MarkdownV2 => MarkdownV2.bold(s),
+            Self::Html => Html.bold(s),
+            Self::Markdown => MarkdownV2.bold(s), // "best effort"
```

Comment:
@Hirrolot what should we do here?

Using `MarkdownV2` formatting may be incorrect. On the other hand `Markdown` is deprecated and many things are can't be expressed in it. Maybe we should just `panic!`?